### ISSUE TITLE
Cannot start nuxt:  nuxt.options.buildModules is not iterable

### ISSUE THREAD

**Issue by howbizarre** [nuxt/framework#9068]
### Environment

Nuxt project info:                                                                                                                                                                  17:35:52

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.18.1`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@8.19.2`
- Builder:          `vite`
- User Config:      `runtimeConfig`, `modules`, `windicss`, `nitro`, `app`, `typescript`
- Runtime Modules:  `nuxt-windicss@2.5.4`, `nuxt-icon@0.1.7`
- Build Modules:    `-`
------------------------------

### Reproduction

>npm outdated
>nuxt     3.0.0-rc.13   3.0.0   3.0.0  node_modules/nuxt  
>npm update
>npm run dev

### Describe the bug

Nuxt show its loading screen only.

### Additional context

When downgrade to 3.0.0-rc.13 all goes normal.
I have the same problem with 3.0.0-rc.14.

### Logs

```shell
Nuxi 3.0.0                                                                                                                                                             
Nuxt 3.0.0 with Nitro 1.0.0                                                                                                                                                         17:31:40
                                                                                                                                                                                
  > Local:    http://localhost:3000/ 
  > Network:  http://192.168.2.129:3000/

ℹ Cleaning up generated nuxt files and caches...                                                                                                                                    

 ERROR  Cannot start nuxt:  nuxt.options.buildModules is not iterable                                                                                                               

  at setup (node_modules/nuxt-windicss/dist/module.mjs:94:23)                                                                                                                       
  at normalizedModule (node_modules/nuxt-windicss/node_modules/@nuxt/kit/dist/index.mjs:614:29)
  at async installModule (node_modules/@nuxt/kit/dist/index.mjs:416:3)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:1825:7)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6779:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6840:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:50:20)
```

**Comment by gravit8** [comment]
I had problems with both RC14 and fresh build of RC13 earlier, adding "@nuxt/kit": "3.0.0-rc.13" to dependencies resolved for now.

**Comment by pi0** [comment]
`buildModules` is not supported in Rc.14 and 3.0.0 releases. `nuxi-windicss` needs uptate. Please track at upstream: https://github.com/windicss/nuxt-windicss/issues/190 (/cc @harlan-zw)

**Comment by acidjazz** [comment]
If you have come here for a temp fix until nuxt-windicss is updated you can find it [here](Temporary nuxt V3 fix for windicss https://github.com/windicss/nuxt-windicss/issues/190#issuecomment-1317838817)

**Comment by TuguldurJ** [comment]
They just released v2.5.5 and it fixed.

**Comment by thermostat42** [comment]
For me, upgrade to NodeJS 18 fixed the issue
